### ISSUE TITLE
GraphQL issue has returned 

### ISSUE THREAD

**Issue by jamescscotlightdirect** [bigcommerce/bigcommerce-for-wordpress#391]
Hello, 

Please refer back to #374 by [jamescscotlightdirect](https://github.com/bigcommerce/bigcommerce-for-wordpress/issues?q=is%3Aissue+author%3Ajamescscotlightdirect) was closed 6 days ago

This looks to be same issue again when Fast headless is enabled. Current import logs listed below. 

[2022-09-26 17:39:47] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":58,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:47] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":59,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:48] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:48] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":59,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:48] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":60,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:49] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:49] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":60,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:49] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":61,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:50] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:50] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":61,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:50] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":62,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:51] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:51] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":62,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:52] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":63,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:53] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:53] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":63,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:53] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":64,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:54] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:54] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":64,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:54] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":65,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:56] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:56] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":65,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:56] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":66,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:58] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:58] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":66,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:58] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":67,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:59] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:39:59] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":67,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:39:59] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":68,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:01] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:01] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":68,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:02] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":69,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:03] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:03] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":69,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:04] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":70,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:05] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:05] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":70,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:05] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":71,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:09] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:09] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":71,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:09] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":72,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:12] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:12] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":72,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:12] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":73,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:13] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:13] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":73,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:13] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":74,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:14] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:14] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":74,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:14] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":75,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:18] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:18] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":75,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:18] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":76,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:19] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:19] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":76,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:20] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":77,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:21] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:21] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":77,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:22] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":78,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:23] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:23] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":78,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:23] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":79,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:25] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:25] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":79,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:25] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":80,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:28] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:28] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":80,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:28] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":81,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:31] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:31] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":81,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:32] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":82,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:34] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:34] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":82,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:34] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":83,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:37] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:37] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":83,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:37] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":84,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:38] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:38] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":84,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:38] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":85,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:45] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:45] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":85,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:45] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":86,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:46] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:46] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":86,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:46] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":87,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:49] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:49] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":87,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:49] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":88,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:50] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:50] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":88,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:50] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":89,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:54] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:54] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":89,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:54] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":90,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:55] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:40:55] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":90,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:40:56] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":91,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:02] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:02] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":91,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:02] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":92,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:03] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:03] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":92,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:03] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":93,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:04] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:04] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":93,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:05] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":94,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:07] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:07] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":94,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:07] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":95,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:10] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:10] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":95,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:11] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":96,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:13] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:13] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":96,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:14] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":97,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:16] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:16] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":97,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:16] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":98,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:16] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:16] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":98,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:17] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":99,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:17] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:17] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":99,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:18] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":100,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:22] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:22] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":100,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:22] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":101,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:25] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:25] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":101,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:25] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":102,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:26] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:26] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":102,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:26] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":103,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:33] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:33] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":103,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:33] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":104,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:40] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:40] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":104,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:40] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":105,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:45] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:45] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":105,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:45] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":106,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:46] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:46] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":106,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:47] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":107,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:48] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:48] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":107,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:48] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":108,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:50] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:50] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":108,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:50] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":109,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:51] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:51] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":109,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:51] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":110,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:55] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:55] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":110,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:55] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":111,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:57] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:57] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":111,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:58] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":112,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:58] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:58] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":112,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:59] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":113,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:59] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:41:59] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":113,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:41:59] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":114,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:01] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:01] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":114,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:02] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":115,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:04] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:04] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":115,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:05] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":116,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:09] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:10] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":116,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:10] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":117,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:11] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:11] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":117,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:12] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":118,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:13] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:13] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":118,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:13] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":119,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:16] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:16] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":119,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:17] BigCommerce.DEBUG: bigcommerce_category import ready for next page of terms {"next":120,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:20] BigCommerce.INFO: Running import task {"state":"updating_categories","description":"Updating Categories"} []
[2022-09-26 17:42:20] BigCommerce.DEBUG: Importing terms for bigcommerce_category taxonomy {"page":120,"limit":2,"taxonomy":"bigcommerce_category"} []
[2022-09-26 17:42:21] BigCommerce.INFO: Running import task {"state":"updated_categories","description":"Updating Brands"} []
[2022-09-26 17:42:21] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":1,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:21] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:22] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:22] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":2,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:22] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":3,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:31] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:31] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":3,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:31] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":4,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:32] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:32] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":4,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:32] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":5,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:36] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:36] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":5,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:36] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":6,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:40] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:40] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":6,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:41] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":7,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:41] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:41] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":7,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:42] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":8,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:45] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:45] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":8,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:46] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":9,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:47] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:47] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":9,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:47] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":10,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:49] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:49] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":10,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:50] BigCommerce.DEBUG: bigcommerce_brand import ready for next page of terms {"next":11,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:52] BigCommerce.INFO: Running import task {"state":"updating_brands","description":"Updating Brands"} []
[2022-09-26 17:42:52] BigCommerce.DEBUG: Importing terms for bigcommerce_brand taxonomy {"page":11,"limit":2,"taxonomy":"bigcommerce_brand"} []
[2022-09-26 17:42:53] BigCommerce.INFO: Running import task {"state":"updated_brands","description":"Fetching products data from the BigCommerce API for channel Scotlight Direct WordPress"} []
[2022-09-26 17:42:54] BigCommerce.ERROR: Could not retrieve the token {"trace":"#0 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(150): BigCommerce\\GraphQL\\BaseGQL->parse_response(Array)\n#1 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(111): BigCommerce\\GraphQL\\BaseGQL->make_request('{\"channel_id\":5...', Array, 'https://api.big...')\n#2 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(57): BigCommerce\\GraphQL\\BaseGQL->request_token()\n#3 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(32): BigCommerce\\GraphQL\\BaseGQL->get_token()\n#4 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/GraphQL_Processor.php(14): BigCommerce\\GraphQL\\BaseGQL->__construct(Object(BigCommerce\\Api\\Configuration))\n#5 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/GraphQL.php(35): BigCommerce\\GraphQL\\GraphQL_Processor->__construct(Object(BigCommerce\\Api\\Configuration), Array)\n#6 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/vendor/pimple/pimple/src/Pimple/Container.php(118): BigCommerce\\Container\\GraphQL->BigCommerce\\Container\\{closure}(Object(Pimple\\Container))\n#7 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(171): Pimple\\Container->offsetGet('bigcommerce.gra...')\n#8 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(304): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}(Object(WP_Term))\n#9 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Task_Manager.php(97): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#10 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(340): BigCommerce\\Import\\Task_Manager->run_next('updated_brands')\n#11 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(307): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#12 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters(NULL, Array)\n#13 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array)\n#14 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Runner/Cron_Runner.php(51): do_action('bigcommerce/imp...', 'updated_brands')\n#15 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(115): BigCommerce\\Import\\Runner\\Cron_Runner->continue_import()\n#16 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(305): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}()\n#17 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)\n#18 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(524): WP_Hook->do_action(Array)\n#19 /home/customer/www/scotlightdirect.co.uk/public_html/wp-cron.php(138): do_action_ref_array('bigcommerce_con...', Array)\n#20 {main}"} []
[2022-09-26 17:42:54] BigCommerce.ERROR: Could not retrieve the token {"trace":"#0 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(150): BigCommerce\\GraphQL\\BaseGQL->parse_response(Array)\n#1 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(85): BigCommerce\\GraphQL\\BaseGQL->make_request('{\"channel_id\":5...', Array, 'https://api.big...')\n#2 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(68): BigCommerce\\GraphQL\\BaseGQL->request_im_token()\n#3 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(33): BigCommerce\\GraphQL\\BaseGQL->get_impersonation_token()\n#4 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/GraphQL_Processor.php(14): BigCommerce\\GraphQL\\BaseGQL->__construct(Object(BigCommerce\\Api\\Configuration))\n#5 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/GraphQL.php(35): BigCommerce\\GraphQL\\GraphQL_Processor->__construct(Object(BigCommerce\\Api\\Configuration), Array)\n#6 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/vendor/pimple/pimple/src/Pimple/Container.php(118): BigCommerce\\Container\\GraphQL->BigCommerce\\Container\\{closure}(Object(Pimple\\Container))\n#7 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(171): Pimple\\Container->offsetGet('bigcommerce.gra...')\n#8 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(304): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}(Object(WP_Term))\n#9 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Task_Manager.php(97): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#10 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(340): BigCommerce\\Import\\Task_Manager->run_next('updated_brands')\n#11 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(307): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#12 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters(NULL, Array)\n#13 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array)\n#14 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Runner/Cron_Runner.php(51): do_action('bigcommerce/imp...', 'updated_brands')\n#15 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(115): BigCommerce\\Import\\Runner\\Cron_Runner->continue_import()\n#16 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(305): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}()\n#17 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)\n#18 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(524): WP_Hook->do_action(Array)\n#19 /home/customer/www/scotlightdirect.co.uk/public_html/wp-cron.php(138): do_action_ref_array('bigcommerce_con...', Array)\n#20 {main}"} []
[2022-09-26 17:42:54] BigCommerce.ERROR: Could not retrieve the token {"trace":"#0 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(150): BigCommerce\\GraphQL\\BaseGQL->parse_response(Array)\n#1 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(111): BigCommerce\\GraphQL\\BaseGQL->make_request('{\"channel_id\":5...', Array, 'https://api.big...')\n#2 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(57): BigCommerce\\GraphQL\\BaseGQL->request_token()\n#3 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(233): BigCommerce\\GraphQL\\BaseGQL->get_token()\n#4 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(270): BigCommerce\\GraphQL\\BaseGQL->get_auth_bearer(false)\n#5 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(130): BigCommerce\\GraphQL\\BaseGQL->get_headers()\n#6 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/GraphQL_Processor.php(58): BigCommerce\\GraphQL\\BaseGQL->make_request(Array)\n#7 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Processors/Headless_Product_Processor.php(71): BigCommerce\\GraphQL\\GraphQL_Processor->request_paginated_products(50)\n#8 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(304): BigCommerce\\Import\\Processors\\Headless_Product_Processor->run()\n#9 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Task_Manager.php(97): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#10 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(340): BigCommerce\\Import\\Task_Manager->run_next('updated_brands')\n#11 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(307): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}('updated_brands')\n#12 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters(NULL, Array)\n#13 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array)\n#14 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Runner/Cron_Runner.php(51): do_action('bigcommerce/imp...', 'updated_brands')\n#15 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(115): BigCommerce\\Import\\Runner\\Cron_Runner->continue_import()\n#16 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(305): BigCommerce\\Container\\Import->BigCommerce\\Container\\{closure}()\n#17 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)\n#18 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(524): WP_Hook->do_action(Array)\n#19 /home/customer/www/scotlightdirect.co.uk/public_html/wp-cron.php(138): do_action_ref_array('bigcommerce_con...', Array)\n#20 {main}"} []
[2022-09-26 17:42:55] BigCommerce.ERROR: GraphQL credentials were missing. No token was sent. [] []
[2022-09-26 17:42:55] BigCommerce.DEBUG: #0 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/BaseGQL.php(150): BigCommerce\GraphQL\BaseGQL->parse_response(Array) #1 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/GraphQL/GraphQL_Processor.php(58): BigCommerce\GraphQL\BaseGQL->make_request('{"query":"query...') #2 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Processors/Headless_Product_Processor.php(71): BigCommerce\GraphQL\GraphQL_Processor->request_paginated_products(50) #3 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(304): BigCommerce\Import\Processors\Headless_Product_Processor->run() #4 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Task_Manager.php(97): BigCommerce\Container\Import->BigCommerce\Container\{closure}('updated_brands') #5 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(340): BigCommerce\Import\Task_Manager->run_next('updated_brands') #6 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(307): BigCommerce\Container\Import->BigCommerce\Container\{closure}('updated_brands') #7 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters(NULL, Array) #8 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array) #9 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Import/Runner/Cron_Runner.php(51): do_action('bigcommerce/imp...', 'updated_brands') #10 /home/customer/www/scotlightdirect.co.uk/public_html/wp-content/plugins/bigcommerce/src/BigCommerce/Container/Import.php(115): BigCommerce\Import\Runner\Cron_Runner->continue_import() #11 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(305): BigCommerce\Container\Import->BigCommerce\Container\{closure}() #12 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array) #13 /home/customer/www/scotlightdirect.co.uk/public_html/wp-includes/plugin.php(524): WP_Hook->do_action(Array) #14 /home/customer/www/scotlightdirect.co.uk/public_html/wp-cron.php(138): do_action_ref_array('bigcommerce_con...', Array) #15 {main} [] []
Webhooks Logs
[2022-09-25 06:00:48] BigCommerce-webhooks.ERROR: Incoming webhook password does not match {"request":{"producer":"stores/9yu28o6y6f","hash":"70dee6f7f667776a1f39541a70081c37390c3aaf","created_at":1664085646,"store_id":"1000986509","scope":"store/product/updated","data":{"type":"product","id":8099}}} []
[2022-09-25 06:00:48] BigCommerce-webhooks.ERROR: Incoming webhook password does not match {"request":{"producer":"stores/9yu28o6y6f","hash":"d4b75fc23fb975c146b14a14a361ce4403939574","created_at":1664085646,"store_id":"1000986509","scope":"store/product/updated","data":{"type":"product","id":8077}}} []
[2022-09-25 06:00:48] BigCommerce-webhooks.ERROR: Incoming webhook password does not match {"request":{"producer":"stores/9yu28o6y6f","hash":"e15901162bd66c737697c7afe428f0fa16cd67d5","created_at":1664085646,"store_id":"1000986509","scope":"store/product/updated","data":{"type":"product","id":8080}}} []
[2022-09-25 06:00:48] BigCommerce-webhooks.ERROR: Incoming webhook password does not match {"request":{"producer":"stores/9yu28o6y6f","hash":"dc3e84ee77582902fe93ae82ae550a65cb001004","created_at":1664085646,"store_id":"1000986509","scope":"store/product/updated","data":{"type":"product","id":8082}}} []
[2022-09-25 06:00:48] BigCommerce-webhooks.ERROR: Incoming webhook password does not match {"request":{"producer":"stores/9yu28o6y6f","hash":"d4b75fc23fb975c1

**Comment by jamescscotlightdirect** [comment]
@yuenvision Hi Paul, Could the team confirm whether if the "Fast Headless" Sync option is not working on a site would this affect the Big Commerce Image CDN or are they completely separate? We are having an issue since moving to the cdn that it is loading large files sizes for images. Once we have compressed these the savings are not being transferred to the front end and the CDN keeps pulling the larger file images. I am struggling to work out whether this would be because this sync option is not currently working and it is not calling the latest information from the API or if it is a completely separate issue and these two are not connected.  Cheers James

**Comment by MlKilderkin** [comment]
Hello @jamescscotlightdirect 
"Fast Headless" option is always pulling images from Bigcommerce CDN. By default it takes standard size of images from product response: `url_standard` Same for standard import with `Import images URLs only` option enabled. Regarding the import problem  it is a bit different. Something is not right with token retrieval. 
Would you please contact Bigcommerce support and provide temporary access details to your WP setup? We will try to investigate and debug the issue. 
Thank you

**Comment by MlKilderkin** [comment]
We'll also investigate images urls retrieving and will try provide some enhancements here

**Comment by jamescscotlightdirect** [comment]
@MlKilderkin Hello thank you for getting back to me. This makes sense now I will open up another ticket on here for the images and provide you with some examples of the URL's on compressed. 

I will also open up a support ticket for the token retrieval and get you access. Do you want me to ask big commerce directly for an email to add in for access or do you want to send it to me?

Thanks again for your help

**Comment by MlKilderkin** [comment]
Hello @jamescscotlightdirect 
Would you please ask a Bigcommerce support for that? They will provide you an email, or you can create a temporary account with temp(fake) email and provide access to them with ticket reference.

Thank you

**Comment by jamescscotlightdirect** [comment]
@MlKilderkin No problem I have opened supoort ticket so should be in touch shortly

**Comment by jamescscotlightdirect** [comment]
@MlKilderkin 
Hello, Just a quick message to say thank you for investigating. We changed the API setup however this still did not work. We then setup a test site on another server and this worked correctly with the new API configuration so it is our server siteground which is blocking the BC API IP addresses it looks like. Thought I would feedback in case you have anyone else with the same problem and changing the API config on it's own does not work. I am waiting on Big Commerce coming back with a IP address list so I can whitelist these. 
Thanks again for your help
James

**Comment by MlKilderkin** [comment]
No problem @jamescscotlightdirect I am glad that the problem is secured
